### ISSUE TITLE
'riverctl spawn' slows computer down

### ISSUE THREAD

**Issue by Ordoviz** [ifreund/river#65]
Running `riverctl spawn program` inserts a bunch of newlines in the terminal and prints the following to river’s log:

```
Segmentation fault at address 0x7f89e69789d0
???:?:?: 0x7f89f117c812 in ??? (???)
```
followed by the program’s output and:
```
/usr/lib/zig/std/dwarf.zig:356:31: 0x246b72 in std.dwarf.parseFormValue (river)
        FORM_strp => FormValue{ .StrPtr = try readAddress(in_stream, endian, is_64) },
                              ^
```

If I run multiple `riverctl spawn` in a short time (e.g. when changing the volume with a key binding) my computer freezes for a few seconds. I use Arch Linux.

Reverting cbf645d468399456eee681d592066fb4afa55169 fixes it.

**Comment by ifreund** [comment]
hmm, I can't reproduce this. I've been running with that commit for about 2 days now and haven't noticed any odd behavior.

An strace log of river running the spawn command would probably be very helpful but might be tedious to obtain.

I'm running void these days but will install arch on a spare drive and check it out when I have a minute.

**Comment by ifreund** [comment]
I was able to reproduce on arch, looking into a fix.

**Comment by ifreund** [comment]
Should be fixed by f3d4e5ac5311f27d8b17f2797a861aab4bae5673, let me know if anything still seems weird.

**Comment by Ordoviz** [comment]
Thanks, this fixes it.
